apps/project/exports/exports.py

```diff
@@ -168,6 +169,7 @@ def _export_project_data(project: Project, tmp_directory: Path):
             "last_contribution_date",
         ),
     )
+    send_slack_message_for_project.delay(project_id=project.id, action="progress-change")
```

Comment:
This should be conditional. We should only send progress messages when notification has not been sent for that progress mark.